TACC-Cloud/geoapi
# task/WG-34: remove DB warnings related to model overlaps

## Overview: ##

Remove DB warnings related to model overlaps

Also updating PR template.

## Related Jira tickets: ##

* [WG-34](https://jira.tacc.utexas.edu/browse/WG-34)

## Testing Steps: ##
1. Run `geoapi` and ensure that warnings to related to model overlap are no longer there.  See Jira ticket for example warnings.

## Notes: ##

TODO
- [x] check if migrations are needed.
